selfservice/strategy/oidc/types.go

```diff
@@ -40,8 +46,10 @@ func NewCredentials(provider, subject string) (*identity.Credentials, error) {
 }
 
 type ProviderCredentialsConfig struct {
-	Subject  string `json:"subject"`
-	Provider string `json:"provider"`
+	Subject               string `json:"subject"`
+	Provider              string `json:"provider"`
+	EncryptedAccessToken  string `json:"encrypted_access_token"`
+	EncryptedRefreshToken string `json:"encrypted_refresh_token"`
 }
```

Comment:
Maybe add initial_id_token too?

Reply:
Some systems issue an ID Token alongside the `access_token`. It will be available as `id_token` if set. Some applications might want to use this to perform additional checks! In general, this refers to OpenID Connect :)

Reply:
ok got it

Reply:
I saw that the id token is a jet that may contain audience and nonce. So that mean we should encrypt it as well